src/Shared/Components/CICDHistory/LogsRenderer.tsx

```diff
@@ -408,10 +440,41 @@ const LogsRenderer = ({
         }
     }, [handleToggleOpenAllStages])
 
+    const handleNextSearchResult = () => {
+        if (searchResults.results.length > 0) {
+            const currentIndex = (searchResults.currentIndex + 1) % searchResults.results.length
+            setSearchResults({
```

Comment:
Can we keep currentIndex as separate state since we are setting searchResults in getStageListFromStreamData, which may be lead to bugs in future, if we miss to batch a state